route/build/cpl/RtmMod.F90

```diff
@@ -249,6 +253,7 @@ SUBROUTINE route_ini(rof_active,flood_active)
     if ((nsrest == nsrContinue) .or. &
         (nsrest == nsrBranch)) then
       call RtmRestGetfile()
+      isRestart=.true.
```

Comment:
Add a else statement to set this to false for the else condition.

Reply:
Added else statement and isRestart is set to false